admin/multiadd.php

```diff
@@ -17,55 +17,76 @@
 	}
 
 	if (mysqli_num_rows ( $series_list ) > 0) {
-		$dropdown = "<select class=\"element select medium\" id=\"element_6\" name=\"series_name\">\n";
-		$dropdown .= "<option value=\"\" selected=\"selected\"></option>\n";
+		$dropdown = '<select class="form-control" id="element_6" name="series_name">';
```

Comment:
id="element_6" is leftover from some form generator I was using when I first started making the site. It can be removed.

I don't remember where it was, but I had an issue using single quotes instead of double quotes. It was probably some concatenation weirdness. As long as it works, I'm fine with single quotes.